### DISCUSSION TITLE
React navigation v5 - v6

### DISCUSSION THREAD

**Discussion by Only-IceSoul** [software-mansion/react-native-screens#1180] [Q&A] [answered]
Does this work with react navigation v5 or v6?

**Comment by kacperkapusciak** [comment, marked as answer]
Both! 

- For `react-navigation v5` import `native-stack` from `react-native-screens/native-stack`
- For `v6` install `@react-navigation/native-stack` and use `native-stack` from there

Part of README that states about that:
https://github.com/software-mansion/react-native-screens#using-createnativestacknavigator-with-react-navigation

`react-native-screens` as a package works with both of these versions. The `native-stack` part is the only thing that changed.
